### PR TITLE
Bug 558/ta leaveslip bug

### PR DESCRIPTION
**Pull request by jontey**
Update object manager for leaveslips to only show current term leaveslips.

### CODE REVIEW

**Comment by jontey** [comment]
Oops. not ready yet. Still need to work on #565

File: ap/leaveslips/models.py [hunk 1]
```diff
@@ -160,18 +175,34 @@ def get_ta_update_url(self):
   def get_update_url(self):
     return reverse('leaveslips:individual-update', kwargs={'pk': self.id})
 
+
+class GroupSlipManager(models.Manager):
```
Comment (billyeh) [inline ap/leaveslips/models.py]: What happens if we just use the same manager for both classes? It's the same code.

Comment (jontey) [reply ap/leaveslips/models.py]: Group Slips have start and end times, for IndividualSlips we use rolls

Comment (billyeh) [reply ap/leaveslips/models.py]: Oh yeah.

**Comment by jontey** [comment]
I think it's good for review
Fixes #558

**Comment by billyeh** [comment]
Yeah, looks good.

**Pull request merged**